packages/apollos-data-prayer/src/prayer-requests/__tests__/resolver.test.js

```diff
@@ -71,56 +119,10 @@ describe('PrayerRequest resolvers', () => {
         }
       }
     `;
-    const responseMock = jest.fn(() =>
-      Promise.resolve([
-        {
-          id: 'PrayerRequest:b36e55d803443431e96bb4b5068147ec',
-          firstName: 'Isaac',
-          lastName: 'Hardy',
-          text: 'Pray this works.',
-          requestedByPersonAliasId: 447217,
-          campusId: 16,
-          categoryId: 2,
-          flagCount: 0,
-          prayerCount: 4,
-          attributeValues: {
-            isAnonymous: {
-              value: 'True',
-            },
-          },
-        },
-        {
-          id: 'PrayerRequest:57c465ee3cd69524d729569b338607de',
-          firstName: 'Rich',
-          lastName: 'Dubee',
-          text: 'Help me',
-          requestedByPersonAliasId: 447217,
-          campusId: 16,
-          categoryId: 2,
-          flagCount: 0,
-          prayerCount: 4,
-          attributeValues: {
-            isAnonymous: {
-              value: 'True',
-            },
-          },
-        },
-      ])
-    );
-
-    context.dataSources.PrayerRequest.get = responseMock;
-    context.dataSources.Person.getFromAliasId = jest.fn(() =>
-      Promise.resolve({
-        id: 1,
-        firstName: 'Isaac',
-        lastName: 'Hardy',
-      })
-    );
 
-    const rootValue = {};
```

Comment:
You could just as easily define this as `rootValue` in the `beforeEach` and leave it as `rootValue` in the parameter list so that everyone knows what it is. I might actually like that better. Then I know what that empty array is supposed to be for. Thoughts?

Reply:
agree